packages/palette/src/elements/Input/Input.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react"
+import styled from "styled-components"
+import { themeGet } from "styled-system"
+import { color, space } from "../../helpers"
+import { Box } from "../Box"
+import { Sans, Serif } from "../Typography"
+
+export interface InputProps extends React.HTMLProps<HTMLInputElement> {
+  description?: string
+  disabled?: boolean
+  error?: string
+  required?: boolean
+  title?: string
+}
+
+/**
+ * Input component
+ */
+export const Input: React.SFC<InputProps> = ({
+  description,
+  disabled,
+  error,
+  required,
+  title,
+  ...rest
+}) => {
+  return (
+    <StyledBox>
+      {title && (
+        <Serif mb="0.5" size="3">
+          {title}
+          {required && <Required>*</Required>}
+        </Serif>
+      )}
+      {description && (
+        <Serif color="black60" mb="1" size="2">
+          {description}
+        </Serif>
+      )}
+      <StyledInput disabled={disabled} error={!!error} {...rest as any} />
+      {error && (
+        <Sans color="red100" mt="1" size="2">
+          {error}
+        </Sans>
+      )}
+    </StyledBox>
+  )
+}
+
+const StyledBox = styled(Box)`
+  width: 100%;
+`
+StyledBox.displayName = "StyledBox"
```

Comment:
Needed? If so, it might be a little more semantic to use something like `<InputContainer>`